phpBB/includes/ucp/ucp_main.php

```diff
@@ -643,7 +693,7 @@ function main($id, $mode)
 	*/
 	function assign_topiclist($mode = 'subscribed', $forbidden_forum_ary = array())
 	{
-		global $user, $db, $template, $config, $cache, $auth, $phpbb_root_path, $phpEx, $phpbb_container;
+		global $user, $db, $template, $config, $cache, $auth, $phpbb_root_path, $phpEx, $phpbb_container, $request, $phpbb_dispatcher;
```

Comment:
Why are you adding `$request` here? Seems like this was caused by one of your first commits.